_posts/2015-06-26-ux-part-1.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: The OME UX Team. Part 1 - Bringing UX to OME
+categories: User Experience
```

Comment:
Needs to be one word or 'use-experience' - system uses a space to mark new categories